merkletree/str.go

```diff
@@ -21,10 +21,10 @@ type SignedTreeRoot struct {
 	PreviousEpoch   uint64
 	PreviousSTRHash []byte
 	Signature       []byte
-	Policies        Policies
+	Policies        *Policies
```

Comment:
Thank for the clarification. Then I'm also slightly in favour of making this a copy instead of a pointer. Simply because it is slightly more explicit.